src/dotnet/APIView/ClientSPA/src/app/_models/userPreferenceModel.ts

```diff
@@ -20,8 +22,10 @@ export class UserPreferenceModel {
         this.hideLeftNavigation = false;
         this.showHiddenApis = false
         this.showDocumentation = false;
-        this.showComments = false;
-        this.showSystemComments = false;
+        this.showComments = true;
```

Comment:
is there a reason why this defaults changed?

Reply:
We want to show comments by default. The defaults here are redundant actually since it will be populated from the user preference from the server.